tests/gabbits/create-snippet.yaml

```diff
@@ -0,0 +1,116 @@
+common:
+  - &datetime_regex /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d+\+\d{2}:\d{2}/
+  - &slug_regex /[a-zA-Z0-9]+/
+
+fixtures:
+  - XSnippetApi
+
+tests:
+  - name: create a new snippet
+    POST: /v1/snippets
+    request_headers:
+      content-type: application/json
+    data:
+      title: Hello, World!
+      syntax: python
+      content: print('Hello, World!')
+      tags:
+        - spam
+        - eggs
+    status: 200
+    response_headers:
+      content-type: application/json
+    response_json_paths:
+      $.id: *slug_regex
+      $.title: Hello, World!
+      $.syntax: python
+      $.created_at: *datetime_regex
+      $.updated_at: *datetime_regex
+      $.tags.`sorted`:
+        - eggs
+        - spam
+      $.changesets.`len`: 1
```

Comment:
According to our API, snippets do not expose changesets. Snippets always return the most recent changeset in its `content` property. We were planning to expose changesets via separate endpoint.

Reply:
Ah, thanks for reminding me! I completely forgot about that.

Ok, I think this is one more reason to have a custom implementation of `Serializable` for `Snippet` rather than derive it. That way we can control the output and have the field `content` populated by the latest changeset. I'll go forward with https://github.com/xsnippet/xsnippet-api/pull/114 then and re-work this.